jekyll/_cci2/api-intro.md

```diff
@@ -0,0 +1,122 @@
+
+With API v2, CircleCI is introducing a string representation of the triplet called the `project_slug`, which takes the following form:
+
+`<project_type>/<org_name>/<repo_name>`
+
+The `project_slug` is included in the payload when pulling information about a project as well as when looking up a pipeline or workflow by ID. The `project_slug` can then be used to get information about the project. It is possible in the future the shape of a `project_slug` may change, but in all cases it would be usable as a human-readable identifier for a given project.
+
+### Authentication
+
+The CircleCI API v2 enables users to be authenticated by simply sending your API token as the username of the HTTP request. For example, if you have set `CIRCLECI_TOKEN` in your shell's environment, then you could then use `curl` with that token like the example shown below:
+
+`curl -user ${CIRCLECI_TOKEN} https://circleci.com/api/v2/me`
+
+### Pipelines
+
+The CircleCI API v2 includes the use of pipelines to assist you in triggering workflows from the CircleCI API. By enabling pipelines, you can take advantage of specific API use cases such as:
+
+- Jobs named `build` will be wrapped in a workflows stanza by the processor.
+- Projects for which auto-cancel is enabled in the **Advanced Settings** will have workflows on non-default branches cancelled when a newer build is triggered on that same branch.
+
+For more detailed information about pipelines and how you can use them in your workflows and builds, please see the [Build Processing](https://circleci.com/docs/2.0/build-processing/)page.
+
+#### Triggering a Pipeline with Parameters Example
+
+Here is a simple example using `curl` to trigger a pipeline with parameters:
+
+```
+curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d '{
+  "parameters": {
+    "myparam": "./myspecialdir",
+    "myspecialversion": "4.8.2"
+  }
+}' https://circleci.com/api/v2/project/${project_slug}/pipeline
+```
+
+In the above example the `project_slug` would take the form `:vcs/:org/:project`. For example, the project slug `gh/CircleCI-Public/circleci-cli` tells `CircleCI` to use the project found in the GitHub organization CircleCI-Public in the repository named `circleci-cli`.
+
+**IMPORTANT** Pipeline parameters are **not** treated as sensitive data and **must not** be used for sensitive values (secrets). You can find this sensitive information in the [Project Settings] page(https://circleci.com/docs/2.0/settings/) and [Contexts](https://circleci.com/docs/2.0/glossary/#context)
+
+## Changes In Endpoints
+
+The CircleCI API v2 release includes several new endpoints, as well as some other endpoints that have been deprecated. The sections below list the endpoints added for this release, in addition to the endpoints that have been removed.
```

Comment:
To me this reads like some deprecated endpoints have been reintroduced. 
```suggestion
The CircleCI API v2 release includes several new endpoints, and deprecates some others. The sections below list the endpoints added for this release, in addition to the endpoints that have been removed.
```

Reply:
Feel free to ignore - It's just the original wording to me sounded like some old api endpoints had been reintroduced.